Wikipedia/Code/EditSummaryViewController.swift

```diff
@@ -0,0 +1,84 @@
+import UIKit
+import WMF
+
+enum EditSummaryViewControllerConstants {
```

Comment:
I like to implement constants [this way](https://github.com/raywenderlich/swift-style-guide#constants).

I tried to find a unified way of creating constants in Swift in your project but couldn't find anything. Did I miss something?